scripts/watch.js

```diff
@@ -0,0 +1,28 @@
+process.env.NODE_ENV = 'development';
+
+const fs = require('fs-extra');
+const paths = require('react-scripts/config/paths');
+const webpack = require('webpack');
+const config = require('react-scripts/config/webpack.config.dev.js');
+
+// removes react-dev-utils/webpackHotDevClient.js at first in the array
+config.entry.shift();
+
+webpack(config).watch({}, (err, stats) => {
+  if (err) {
+    console.error(err);
+  } else {
+    copyPublicFolder();
+  }
+  console.error(stats.toString({
+    chunks: false,
+    colors: true
+  }));
+});
+
+function copyPublicFolder() {
+  fs.copySync(paths.appPublic, paths.appBuild, {
+    dereference: true,
+    filter: file => file !== paths.appHtml
+  });
+}
```

Comment:
@MartyTheeMartian @Moe-Shoman Maybe I should just remove `watch.js` entirely?  Is there any advantage to keeping it now that the Node dev server is proxying through to Go?